parachains/runtimes/contracts/contracts-rococo/src/lib.rs

```diff
@@ -99,6 +99,11 @@ pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, Si
 
 pub type Migrations = (pallet_contracts::Migration<Runtime>,);
 
+type EventRecord = frame_system::EventRecord<
+	<Runtime as frame_system::Config>::RuntimeEvent,
+	<Runtime as frame_system::Config>::Hash,
+>;
```

Comment:
should this alias be defined in the pallet_contracts?

Reply:
In pallet_contracts this is generic over T https://github.com/paritytech/substrate/blob/jg/12412-contracts-return-emitted-events-for-dry-runs/frame/contracts/src/lib.rs#L151
This way we leave it open to the implementor to configure it.